pkg/state/filter.go

```diff
@@ -30,7 +30,11 @@ func FilterOut(currentState shared.State) (shared.State, error) {
 }
 
 func filterOutRoutes(kind string, state map[string]interface{}, interfacesFilterGlob glob.Glob) {
-	routes := state["routes"].(map[string]interface{})
+	routesRaw, hasRoutes := state["routes"]
+	if !hasRoutes {
+		return
+	}
+	routes := routesRaw.(map[string]interface{})
```

Comment:
Why don't you test whether the casting went right here, but you do check it further down?

Reply:
In the case of routes there is always "config" and "running"

Reply:
How is it relevant to casting? Do you mean that in case of "bridge" for instance we can get nil?

Reply:
Ahh right, misunderstood, added it.